<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VhallRenderMaskWdg</class>
 <widget class="QWidget" name="VhallRenderMaskWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>249</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>28</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VhallRenderMaskWdg</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="pageTrophyWdg" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="TrophyWdg" name="pTrophyWdg" native="true">
        <property name="minimumSize">
         <size>
          <width>94</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>94</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>91</width>
          <height>127</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="TeacherCtrlWdg" name="mTeacherCtrlWdg" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>28</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_userName">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 14px &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_pen">
        <property name="minimumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/ctrl/img/ctrl/small_pen.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_noMic">
        <property name="minimumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/ctrl/img/ctrl/mute_mic_small_notice.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_StuNoMic">
        <property name="minimumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/ctrl/img/ctrl/mute_mic_small_notice.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labStatSpacer">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TrophyWdg</class>
   <extends>QWidget</extends>
   <header>trophywdg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TeacherCtrlWdg</class>
   <extends>QWidget</extends>
   <header>TeacherCtrlWdg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CWidget</class>
   <extends>QWidget</extends>
   <header>cwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../VLive/VLive.qrc"/>
 </resources>
 <connections/>
</ui>
